Pipelines/ci.yaml

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# CI build producing developer packages.
+
+variables:
+- template: Config/settings.yaml
+- name: Version.Revision
+  value: $[counter(variables['MRTKGraphicsToolsVersion'], 1)]
+- name: Version.PreReleaseRevision
+  value: $[counter(variables['MRTKGraphicsToolsReleaseTag'], 1)]
+
+resources:
+  repositories:
+  - repository: PipelineTools
+    type: git
+    endpoint: ToolsAccess
+    name: tools.internal
+    ref: 78ecf591964ecf511b4c7a0cd84d98ce11e8be16
+
+pr: none # disable PR validation
+trigger:
+  tags:
+    include:
+    - '*'
+  branches:
+    exclude:
+    - main
```

Comment:
Sorry I'm not familiar with this syntax. Does this mean these branches are not considered for the trigger?

Reply:
Yes, my goal is to exclude main from the trigger so it doesn't run for every merge to main.

Reply:
Oh okay, that makes sense. I wonder if we need to exclude "all branches" and only look for when tags change?